client/src/components/App.js

```diff
@@ -37,15 +37,15 @@ const App = (props) => {
       <TopBar user={currentUser} />
       <Switch>
         <Route exact path="/">
-          <h2>Hello from react</h2>
+          <h2 className="welcome-header">Pod Classed: Classify your Pod</h2>
```

Comment:
haha minor grammar style edit: "Your" should be upcased, since this is a title and you've upcased P in Pod